### PR TITLE
Make watcher skip moly points

### LINKED ISSUE #112: Monitoring an ongoing MIBI run notebook generates error if molly points present
**Describe the bug**
molly QC points generates the following error 

KeyError: 'Found unexpected bin file, D:\\Data\\2022-05-21_TA586_9B_Run7\\fov-4-scan-1.bin...'

**To Reproduce**
1. Go to 3a_monitor_MIBI_run notebook
2. Set run name to 2022-05-21_TA586_9B_Run7
3. Callback initialization and start watcher script
4. Actual FOVs will show "discovered fov-1-scan-1, beginning per-fov callbacks..."  but will generate an error message when it gets to the molly FOV. 

**Expected behavior**
any molly points found in the run json file should be skipped

**Screenshots**
Screen Shot 2022-05-27 at 4 00 47 PM

### PR DESCRIPTION
**Pull request by ackagel**
Closes #112 .

For a given scanCount in the run json, only `fov-{runOrder}-scan-{scanCount}` was expected, when all values ≥0 and ≤scanCount should be expected.  This was corrected so that all scans are expected.

Further, moly points are identified via the `"standardTarget"` field with value `"Molybdenum Foil"`, and added to a list.  All points are checked against this list, and if they are members, are properly skipped, without raising an error message.

### CODE REVIEW

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, just needs a test case

**Comment by camisowers** [comment]
Creates a combined run dir with both moly and regular fovs to test watcher on rather than using the tissue dir. Also removes the moly case testing from watcher_callbacks_test.py

**Review by ngreenwald — CHANGES_REQUESTED** [review]

File: toffy/fov_watcher_test.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 from toffy.watcher_callbacks import build_callbacks
 from toffy.json_utils import write_json_file
 
-TISSUE_DATA_PATH = os.path.join(Path(__file__).parent, 'data', 'tissue')
+COMBINED_DATA_PATH = os.path.join(Path(__file__).parent, 'data', 'combined')
```
[the remark this answers is not in the document]
Comment (camisowers) [reply toffy/fov_watcher_test.py:25]: Not for watcher, but there are a few other test files that do. I can just rework those to use the combined directory.

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, tests are failing

**Review by ngreenwald — APPROVED** [review]

**Pull request merged**